### PR TITLE
Add Tubular

### PR DESCRIPTION
**Pull request by geoperez**

### CODE REVIEW

**Comment by thednp** [comment]
from my experience, you should delete these last 2 commits, delete your fork, and redo everything again, you can only commit ONCE those 2 files, see my last merged commit here.

**Comment by geoperez** [comment]
Really? The Doc says: "If your PR exceeds 2 commits(1 per file) it will be declined." I'm just using two. 

Do I need to download the repo and squash them?

**Comment by thednp** [comment]
I remember the admin here said something about squash, so maybe, but now, I do as I told you, and done right, it'll never fail.

**Comment by geoperez** [comment]
I need to leave now, let's see if admin has any comment meanwhile.

Thank you.

**Comment by jimaek** [comment]
Please include both files https://github.com/jsdelivr/jsdelivr/pull/4339 in 1 PR. No need for a PR per file

**Comment by geoperez** [comment]
I updated this PR with a new commit, is it OK? Or I need to clone and squash them?

**Comment by thednp** [comment]
@geoperez you need to create both files then commit both in ONE PR, as the admin said, so, DELETE your entire fork repository, create BOTH files again and PR, as I said previously.

**Comment by geoperez** [comment]
Hi @thednp, is this really necessary? The clone will take like forever :smile:

**Comment by thednp** [comment]
Well, it's better than waiting to get approved other way.

File: files/tubular/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "tubular",
+  "repo": "unosquare/tubular",
+  "files": {
+    "include": ["tubular-bundle.js", "tubular-bundle.min.js", "tubular-bundle.css","tubular-bundle.min.css"]
```
Comment (jimaek) [inline files/tubular/update.json]: You need to add a basePath parameter pointing to "dist/" https://github.com/unosquare/tubular/tree/master/dist

Also you dont have any tagged versions, our bot wont work until you start tagging them

**Comment by geoperez** [comment]
@jimaek I added the tag version and updated the update.json with basePath.

**Comment by geoperez** [comment]
BTW @jimaek the update.json sample in your documentation (https://github.com/jsdelivr/jsdelivr/blob/master/CONTRIBUTING.md) doesn't have the basePath.

**Comment by jimaek** [comment]
basePath is not a required parameter, thats why its not there. But in your case its needed.

File: files/tubular/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "tubular",
+  "repo": "unosquare/tubular",
+  "files": {
+    "basePath": "dist/",
+    "include": ["tubular-bundle.js", "tubular-bundle.min.js", "tubular-bundle.css","tubular-bundle.min.css"]
```
Comment (jimaek) [inline files/tubular/update.json]: remove the unminified files and then either squash your commits or create a new PR.
The max amount of commits is 2 per PR

**Comment by geoperez** [comment]
Got it, thanks. So now what I need to do?

**Comment by geoperez** [comment]
I removed the unminified files. I'll recreate the PR.

**Comment by geoperez** [comment]
Closing this, new PR #4346

**Pull request closed without merging**